### DISCUSSION TITLE
lib files missing

### DISCUSSION THREAD

**Discussion by MungaiKen** [Cyfrin/foundry-full-course-cu#1362] [General]
'''
import {Script} from "forge-std/Script.sol";
''''
cannot find the forge-std/Script.sol file, how do I go about this?

**Comment by xyz899** [comment]
`forge install foundry-rs/forge-std`

restart your terminal and VsCode it should work

**Reply by MungaiKen** [reply]
'''Installing forge-std in /home/mungaiken/new_folder/ecoin/lib/forge-std (url: Some("https://github.com/foundry-rs/forge-std"), tag: None)
Error: 
The target directory is a part of or on its own an already initialized git repository,
and it requires clean working and staging areas, including no untracked files.

Check the current git repository's status with `git status`.
Then, you can track files with `git add ...` and then commit them with `git commit`,
ignore them in the `.gitignore` file, or run this command again with the `--no-commit` flag.
'''

**Reply by xyz899** [reply]
`forge install foundry-rs/forge-std --no-commit`

**Reply by MungaiKen** [reply]
Thank you very much
Its sorted now

**Comment by PatupaiarehePunk** [comment]
I am also having this problem. 

I foundry's latest release it does not seem to include any solidity files under *forge-std*.

**Reply by MungaiKen** [reply]
forge install foundry-rs/forge-std --no-commit

**Reply by PatupaiarehePunk** [reply]
I ran that command successfully:

```
╭─alexbissell@alexs-mbp ~/Solidity/foundry-f23/foundry-simple-storage-f23 ‹master› 
╰─$ forge install foundry-rs/forge-std --no-commit
Installing forge-std in /Users/alexbissell/Solidity/foundry-f23/foundry-simple-storage-f23/lib/forge-std (url: Some("https://github.com/foundry-rs/forge-std"), tag: None)
    Installed forge-std v1.7.6
```

I resolved this by changing my directory to my git directory that I set up before the tutorial warned against it.  

in my instance it was a matter of hitting:

```zsh
cd hello_foundry
forge install foundry-rs/forge-std --no-commit
```

One thing that I found confusing is that there is no file called script in ***lib/forge-std***. 
I suspect there is an init for the file that changes the directory so the user doesn't have to specify the path every time for all of the scripts.
